test/e2e_node/kubelet_test.go

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e_node_test
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/golang/glog"
+	. "github.com/onsi/ginkgo"
+	//	. "github.com/onsi/gomega"
```

Comment:
To make uncommenting it easier.  It is from the ginkgo tutorial and will be used as the matcher validation library once real tests are written.  I also don't like commented code :).  Can delete it for now and add it back when needed.